deploy/olm-catalog/service-telemetry-operator/manifests/service-telemetry-operator.clusterserviceversion.yaml

```diff
@@ -206,6 +206,52 @@ spec:
     mediatype: image/svg+xml
   install:
     spec:
+      clusterPermissions:
```

Comment:
Mostly curious, but these were generated via `operator-sdk generate bundle` after adding the `cluster_role.yaml` and `cluster_role_bindings.yaml` to the `deploy/` directory?

Reply:
Just realized you put these in the role and role_binding.yaml files. You can actually break these out to `cluster_role.yaml` and `cluster_role_binding.yaml` files, and then run the `operator-sdk generate bundle` and it will populate the CSV for you.

I just did a local test to make sure I wasn't mis-remembering, but it does indeed populate the CSV that way.